src/Build/project.json

```diff
@@ -3,6 +3,8 @@
     "net46": {
       "dependencies": {
         "Microsoft.VisualStudio.Setup.Configuration.Interop": "1.14.114",
+        "NuGet.Commands": "4.5.0",
+        "NuGet.Protocol": "4.5.0",
```

Comment:
Is the Newtonsoft.Json reference coming to us transitively through this? Since we use it explicitly I'd like to see it mentioned explicitly.

Reply:
Yes, it comes through [NuGet.ProjectModel](https://www.nuget.org/packages/NuGet.ProjectModel/)

Technically the version doesn't matter since we only load it from where NuGet is at.  We only have one call which I highly doubt would ever be broken.  But if you feel strongly about it...